plugins/micron/micron-utils.c

```diff
@@ -0,0 +1,276 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+/* Copyright (c) Micron, Inc 2024.
+ * 
+ * @file: micron-utils.h
+ * @brief: This module contains all the utilities needed for micron nvme plugin and other micron modules.
+ * @author: Chaithanya Shoba <ashoba@micron.com>
+ */
+
+#include "micron-utils.h"
+
+int hex_to_int(char c)
+{
+    if (c >= '0' && c <= '9')
+        return c - '0';
+    else if (c >= 'A' && c <= 'F')
+        return 10 + (c - 'A');
+    else if (c >= 'a' && c <= 'f')
+        return 10 + (c - 'a');
+    else
+        return -1; // Invalid character
+}
+
+char* hex_to_ascii(const char* hex) 
+{
+    int hexLength = strlen(hex);
```

Comment:
The camel case variable name hexLength should not be used as below. Also should be used the tab space indentation.
```diff
-     int hexLength = strlen(hex);
+ 	int hex_length = strlen(hex);
```

Reply:
Done